### ISSUE TITLE
Advancements such as Helmet,Boots,Chestplate anad Leggings keep spamming even before the player joins. New advancements get sent only once but those advancements get spammed randomly in the game

### ISSUE THREAD

**Issue by GTDEScripts** [DiscordSRV/DiscordSRV#1605]
### Expected behavior

I expect the plugin to only send the advancements once

### Actual behavior

No idea

### Steps to reproduce

It triggers when you remove armour,chestplate boots and leggings or put them back on

### Server software and version

1.20.2

### Checks

- [X] I am __not__ using an outdated version of DiscordSRV.
- [X] I asked in [DiscordSRV's Discord server](https://discord.discordsrv.com) to see whether this issue is in fact a bug that needs to be fixed.

### Anything else

It only started happening after I added a datapack which has new advancements

**Comment by GTDEScripts** [comment]
[20:17:33] [Craft Scheduler Thread - 14/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 14/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 16/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 16/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 14/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 15/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 14/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 17/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 15/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 17/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 16/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 16/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 7/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 7/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 17/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 17/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 18/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:33] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:34] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:34] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:34] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:34] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:34] [Craft Scheduler Thread - 18/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:34] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/37aad4cd3e314f99b47c37765b481105.png?size=128&overlay#32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:34] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 37aad4cd-3e31-4f99-b47c-37765b481105. The texture is: 32b2c3734b6d2bc0c5eed5c4e72efa948062b58d1a85a820b8d87c2c058be197
[20:17:43] [Server thread/INFO]: Named entity GlowSquid[' ✦ Glow Squid✦  ❤2/10❤'/2417, l='ServerLevel[world]', x=-1172.58, y=2.20, z=1339.40] died:  ✦ Glow Squid✦  ❤2/10❤ was slain by Axolotl
[20:17:56] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/7338deb4103348e081886d2475ed5fbc.png?size=128&overlay#43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:17:56] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 7338deb4-1033-48e0-8188-6d2475ed5fbc. The texture is: 43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98
[20:18:03] [Craft Scheduler Thread - 17/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/7338deb4103348e081886d2475ed5fbc.png?size=128&overlay#43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:18:03] [Craft Scheduler Thread - 17/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 7338deb4-1033-48e0-8188-6d2475ed5fbc. The texture is: 43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98
[20:18:25] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/7338deb4103348e081886d2475ed5fbc.png?size=128&overlay#43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:18:25] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 7338deb4-1033-48e0-8188-6d2475ed5fbc. The texture is: 43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98
[20:18:47] [Server thread/INFO]: Named entity GlowSquid[' ✦ Glow Squid✦  ❤2/10❤'/2415, l='ServerLevel[world]', x=-1159.34, y=6.20, z=1344.85] died:  ✦ Glow Squid✦  ❤2/10❤ was slain by Axolotl
[20:18:59] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/7338deb4103348e081886d2475ed5fbc.png?size=128&overlay#43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:18:59] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 7338deb4-1033-48e0-8188-6d2475ed5fbc. The texture is: 43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98
[20:19:07] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Constructed avatar url: https://crafatar.com/avatars/7338deb4103348e081886d2475ed5fbc.png?size=128&overlay#43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98 from https://crafatar.com/avatars/{uuid-nodashes}.png?size={size}&overlay#{texture}
[20:19:07] [Craft Scheduler Thread - 12/INFO]: [DiscordSRV] [UNCATEGORIZED DEBUG] Avatar url is for uuid: 7338deb4-1033-48e0-8188-6d2475ed5fbc. The texture is: 43c9af51fe4e9a373f99a35d9b03e2e98cd9056fa6bfaa92e15c81a61ff32b98

**Comment by Dinty1** [comment]
Duplicate of #1601

**Comment by granny** [comment]
Duplicate of #1575
